test/Chirp.Infrastructure.Tests/RepositoryTests/AuthorRepositoryTest.cs

```diff
@@ -0,0 +1,261 @@
+                Text = "TestCheep" + i,
+                AuthorDto = authorDto
+            };
+
+            context.Authors.Add(authorDto);
+            context.Cheeps.Add(cheepDto);
+        }
+
+        context.SaveChanges();
+
+        //Act
+        AuthorDTO expectedAuthor = theAuthor;
+        AuthorDTO returnedAuthor = authorRepository.GetAuthorByName("TestAuthor5");
+
+        //Assert
+        Assert.Equal(expectedAuthor, returnedAuthor);
+
+    }
+
+    [Fact]
+    public void GetCheepsByAuthor_ShouldReturnCorrectCheeps()
+    {
+
+        //Arrange
+        var authorRepository = new AuthorRepository(context);
+
+        AuthorDTO authorDto1 = new AuthorDTO
+        {
+            AuthorId = Guid.NewGuid(),
+            Name = "TestAuthor",
+            Email = "mock@email.com"
+        };
+        CheepDTO cheepDto1 = new CheepDTO
+        {
+            CheepId = Guid.NewGuid(),
+            AuthorId = authorDto1.AuthorId,
+            Text = "TestCheep",
+            AuthorDto = authorDto1
+        };
+
+        context.Authors.Add(authorDto1);
+        context.Cheeps.Add(cheepDto1);
+
+        context.SaveChanges();
+
+        //Act
+        ICollection<CheepDTO> expectedCheep = new List<CheepDTO>();
+        expectedCheep.Add(cheepDto1);
+
+        ICollection<CheepDTO> returnedCheep = authorRepository.GetCheepsByAuthor(authorDto1.Name, 0);
+
+        //Assert
+        Assert.Equal(expectedCheep, returnedCheep);
+    }
+
+    [Fact]
+    public void addAuthor_ShouldAddAuthorToDatabase()
+    {
+        //Arrange
+        var authorRepository = new AuthorRepository(context);
+
+        AuthorDTO authorDto1 = new AuthorDTO
+        {
+            AuthorId = Guid.NewGuid(),
+            Name = "TestAuthor",
+            Email = "mock@email.com"
+        };
+        CheepDTO cheepDto1 = new CheepDTO
+        {
+            CheepId = Guid.NewGuid(),
+            AuthorId = authorDto1.AuthorId,
+            Text = "TestCheep",
+            AuthorDto = authorDto1
+        };
+
+        context.Authors.Add(authorDto1);
+        context.Cheeps.Add(cheepDto1);
+
+        context.SaveChanges();
+
+        // Act
+        int initialAuthorCount = context.Authors.Count();
+
+        AuthorDTO authorDto2 = new AuthorDTO
+        {
+            AuthorId = Guid.NewGuid(),
+            Name = "TestAuthor2",
+            Email = "TestEmail2@test.com"
+        };
+        CheepDTO cheepDto2 = new CheepDTO
+        {
+            CheepId = Guid.NewGuid(),
+            AuthorId = authorDto2.AuthorId,
+            Text = "TestCheep2",
+            AuthorDto = authorDto2
+        };
+
+        authorRepository.AddAuthor(authorDto2);
+        context.Cheeps.Add(cheepDto2);
+
+        context.SaveChanges();
+
+        int updatedAuthorCount = context.Authors.Count();
+
+        // Assert
+        Assert.Equal(initialAuthorCount + 1, updatedAuthorCount);
+    }
+
+    [Fact]
+    public void GetAuthorById_ShouldReturnCorrectAuthor()
```

Comment:
All the comments in the first test also apply here

Reply:
This should also be fixed now!